invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/components/DocumentCard/DocumentCard.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { Card, Image } from 'semantic-ui-react';
+
+export default class DocumentCard extends Component {
```

Comment:
Looks like this component could be converted into a functional component or a `React.PureComponent`. What do you think?

Reply:
I am not sure that we do all the necessary things correct in order to take advantage of the PureComponent. But definitely is something that we need to assess in the end for boosting the performance:) But functional component, even for now doesn't have an impact in React's performance can be a nice refactor, but we need to keep it consistent among the codebase!